roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -1039,7 +1039,9 @@ media_store_path: "/matrix-media-store-parent/{{ matrix_synapse_media_store_dire
 #    config:
 #       directory: /mnt/some/other/directory
 
+{% if matrix_synapse_media_storage_provider_s3_enabled %}
```

Comment:
This `if` is unnecessary. It's also undersireable even.

With our new setup, people should be able to inject media storage providers manually into `matrix_synapse_media_storage_providers`.

Enabling `matrix_synapse_media_storage_provider_s3_enabled` makes the playbook automatically inject a storage provider there, but manual management is also possible.

Reply:
yeah got it removed.